context/app/routes_main.py

```diff
@@ -39,6 +40,10 @@ def details(type, uuid):
     client = _get_client()
 
     entity = client.get_entity(uuid)
+    actual_type = entity['entity_type'].lower()
+    if type != actual_type:
```

Comment:
Something I just noticed, but isn't `type` a protected keywork in python?

Reply:
Good catch.